### PR TITLE
gui

### PR DESCRIPTION
**Pull request by s23deepak**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/s23deepak/s23deepak.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : Gandhi Institute of Technology and Management, Visakhapatnam
```
Comment (CLiu13) [inline _data/s23deepak/s23deepak.md:3]: The institution cannot be longer than 58 characters (you currently have 60 characters).

File: _data/s23deepak/s23deepak.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : Gandhi Institute of Technology and Management, Visakhapatnam
+profile_pic : deepak.png
```
Comment (CLiu13) [inline _data/s23deepak/s23deepak.md:4]: The profile picture path does not match the name of the actual file (```s23deepak.jpg```).

**Comment by s23deepak** [comment]
> 
> 
> Your image file name doesnt match the name in your .md file.

**Comment by s23deepak** [comment]
I have changed the profile picture path as suggested

**Comment by CLiu13** [comment]
Thanks for the changes - please address the institution name character length (see my earlier review).

File: _data/s23deepak/s23deepak.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : Gandhi Institute of Technology and Management, Visakhapatnam
+profile_pic : s23deepak.png
```
Comment (CLiu13) [inline _data/s23deepak/s23deepak.md:4]: Thanks for the changes - however, the file extension is incorrect (your image is a ```.jpg``, not a ```.png```).

Comment (s23deepak) [reply _data/s23deepak/s23deepak.md]: Thanks for the suggestions, I have changed them.

File: _data/s23deepak/s23deepak.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : GITAM University, Visakhapatnam
+profile_pic : s23deepak.jpg
+qoute : Hard work without perseverance is a waste
```
Comment (CLiu13) [inline _data/s23deepak/s23deepak.md:5]: Just noticed - this is likely the missing header (misspelling of ```quote```).

**Comment by s23deepak** [comment]
I have corrected the spelling, but I am unable to create new pull request.

**Comment by s23deepak** [comment]
I have corrected the spelling mistake, but I am unable to create a new pull request.
________________________________

**Pull request closed without merging**